atlp-rwanda/rca-phantom-team1-fe
# chore: configured testing environment, setup github actions

## What does this PR do
- Configured Testing environment with Jest & Enzyme or Testing Library to ensure that all functionalities are tested before deployment.
- GitHub Actions is configured as a continuous integration service used to build and test the app.
- Configured Heroku/Vercel/Netlify Pipeline
- GitHub Actions badge has been added to your repo
- Configure CodeClimate for code analysis and Maintainability checking.

## Description
- This PR configures a continuous integration pipeline using GitHub Actions and Heroku. The continuous integration service should automatically run tests and other preliminary build steps required to ship the next iteration of the product

## How to manually test this 
- Check in the branch `chore/CI-and-QA-setup`
- Run the `yarn test` script to see if the tests are configured
- Go to actions on GitHub
- Check for GitHub actions that are running
- Go to codeClimate to check for maintenance

## Relevant Trello story 
[https://trello.com/c/aUvqt1WS/27-setup-continuous-integration-quality-assurance-pipeline](url)

## Screenshots
![image](https://user-images.githubusercontent.com/53254885/233054015-5af4b23d-3f96-459f-a499-4ea77ad6fafc.png)
![image](https://user-images.githubusercontent.com/53254885/233827941-b274fcc2-e942-48c4-b181-101d40a96fc2.png)
